php/WP_CLI/Runner.php

```diff
@@ -534,6 +534,12 @@ private static function back_compat_conversions( $args, $assoc_args ) {
 					$assoc_args['orderby'] = 'post__in';
 					$assoc_args['field'] = 'url';
 					break;
+				case 'term':
+					$term_ids = array_slice( $args, 3 );
+					$args = array( 'term', 'list', $args[2] );
+					$assoc_args['include'] = implode( ',', $term_ids );
```

Comment:
> I am not sure what is the best path here.

I think this is why I didn't get around to fully implementing the feature previously. I'll give it some thought.